rpcserver.go

```diff
@@ -1923,6 +1923,8 @@ out:
 
 				break out
 			}
+		case <-updateStream.Context().Done():
```

Comment:
Would be great to also apply these changes to the subservers in `lnrpc/...`, for example `lnrpc/routerrpc/router_server.go`.